### PR TITLE
do not print out  users' apikey

### PR DESCRIPTION
**Pull request by steven0711dong**
Fixes #

## Proposed Changes

-
-
-

**Release Note**

```release-note

```

**Docs**

### CODE REVIEW

File: pkg/common/client/config.go [hunk 1]
```diff
@@ -265,9 +264,6 @@ func (b *configBuilder) Build(ctx context.Context) (*sarama.Config, error) {
 		config.ClientID = b.clientId
 	}
 
-	logger := logging.FromContext(ctx)
-	logger.Infof("Built Sarama config: %+v", config)
```
Comment (aliok) [inline pkg/common/client/config.go:269]: Can't we have the message without the config?

Comment (matzew) [reply pkg/common/client/config.go:269]: yeah, would be handy - and perhaps... in DEBUG ? :smile:

Comment (aliok) [reply pkg/common/client/config.go:269]: Ok, then, and info message that the config is built, but the actual config in a debug message?

Comment (lionelvillard) [reply pkg/common/client/config.go:269]: even in DEBUG I don't think we want to print the user apikeys. What about printing the config before the apikey is set? Like maybe set it first to `XXX` and then substitute after printing the config? 

A better solution would be the override the String() function (haven't tried but should work)

Comment (steven0711dong) [reply pkg/common/client/config.go:269]: I can try override the String function

**Review by lionelvillard — APPROVED** [review]

**Pull request merged**